modules/unitbar.lua

```diff
@@ -62,22 +62,6 @@ function GladiusEx:NewUnitBarModule(name, defaults_arena, defaults_party)
 
 			self.frame[unit].statusbar:SetStatusBarColor(color.r, color.g, color.b, color.a or 1)
 			self.frame[unit].icon:SetTexture([[Interface\Glues\CharacterCreate\UI-CharacterCreate-Classes]])
-
-			-- V: wtf is "CLASS_BUTTONS"?
-			if not CLASS_BUTTONS then

```

Comment:
This won't work if you just remove all of this. Looking at how it is done in classicon.lua, CLASS_BUTTONS was renamed to CLASS_ICON_TCOORDS.